build.py

```diff
@@ -62,95 +62,175 @@ def set_font_metaData(font, sort):
     font.info.openTypeOS2WinAscent = 2226
     font.info.openTypeOS2WinDescent = abs(font.info.openTypeOS2TypoDescender)
 
-    if sort != "otf":
-        font.info.openTypeGaspRangeRecords =[
-            {
-                "rangeMaxPPEM" : 9,
-                "rangeGaspBehavior" : [1,3]
-            },
-            {
-                "rangeMaxPPEM" : 50,
-                "rangeGaspBehavior" : [0,1,2,3]
-            },
-            {
-                "rangeMaxPPEM" : 65535,
-                "rangeGaspBehavior" : [1,3]
-            },
-        ]
+    font.info.openTypeGaspRangeRecords = [
+        {"rangeMaxPPEM": 9, "rangeGaspBehavior": [1, 3]},
+        {"rangeMaxPPEM": 50, "rangeGaspBehavior": [0, 1, 2, 3]},
+        {"rangeMaxPPEM": 65535, "rangeGaspBehavior": [1, 3]},
+    ]
 
-def overlapFlag(varFont):
-    
-    tt = varFont
-    glyf = tt["glyf"]
 
+def set_overlap_flag(varfont: fontTools.ttLib.TTFont) -> fontTools.ttLib.TTFont:
+    glyf = cast(_g_l_y_f.table__g_l_y_f, varfont["glyf"])
     for glyph_name in glyf.keys():
         glyph = glyf[glyph_name]
-        # Set OVERLAP_COMPOUND bit for compound glyphs
         if glyph.isComposite():
+            # Set OVERLAP_COMPOUND bit for compound glyphs
             glyph.components[0].flags |= 0x400
-        # Set OVERLAP_SIMPLE bit for simple glyphs
         elif glyph.numberOfContours > 0:
+            # Set OVERLAP_SIMPLE bit for simple glyphs
             glyph.flags[0] |= 0x40
-    return tt
 
+    return varfont
+
+
+def prepare_fonts(
+    designspace: fontTools.designspaceLib.DesignSpaceDocument, name: str
+) -> None:
+    designspace.loadSourceFonts(ufoLib2.Font.open)
+    for source in designspace.sources:
+        if "Mono" in name and "PL" in name:
+            step_set_feature_file(FEATURES_DIR / "features_mono_PL.fea", source.font)
+            print(f"[{name} {source.styleName}] Merging PL glyphs")
+            step_merge_glyphs_from_ufo(
+                NERDFONTS_DIR / "NerdfontsPL-Regular.ufo", source.font
+            )
+            step_set_font_name(name, source.font)
+        elif "Mono" in name:
+            step_set_feature_file(FEATURES_DIR / "features_mono.fea", source.font)
+            step_set_font_name(name, source.font)
+        elif "PL" in name:
+            step_set_feature_file(FEATURES_DIR / "features_code_PL.fea", source.font)
+            print(f"[{name} {source.styleName}] Merging PL glyphs")
+            step_merge_glyphs_from_ufo(
+                NERDFONTS_DIR / "NerdfontsPL-Regular.ufo", source.font
+            )
+            step_set_font_name(name, source.font)
+        else:
```

Comment:
You might want to make this an explicit

```
if name == "Cascadia Code":
    ...
elif ...
else:
    raise ValueError(f"Unknown font configuration: {name}")
```

Because this will catch typos.

Reply:
Ah yeah. I forgot to do that :x

Reply:
done